lib/clipboard.zsh

```diff
@@ -0,0 +1,70 @@
+# System clipboard integration
+#
+# This file has support for doing system clipboard copy and paste operations
+# from the command line in a generic cross-platform fashion.
+#
+# On OS X and Windows, the main system clipboard or "pasteboard" is used. On other
+# Unix-like OSes, this considers the X Windows CLIPBOARD selection to be the
+# "system clipboard", and the X Windows `xclip` command must be installed.
+#
+# When using X Windows, copied data will only be good for a single paste
```

Comment:
Can you elaborate on this? This is not a behavior I can reproduce on my system, I can paste multiple times the same clipboard.